docs/commands.md

```diff
@@ -408,7 +408,7 @@ Starts the server that communicates with connected devices
 
 Specify port to listen on
 
-#### `--projectRoot [list]`
+#### `--projectRoot [path]`
```

Comment:
```suggestion
#### `--projectRoot [string]`
```

Reply:
I chose path as it’s more specific and was used down below on cert and key options.